draft-ppm-protocol.md

```diff
@@ -548,11 +559,10 @@ struct {
   input share.
 * `payload` is the encrypted input share.
 
-To generate the report, the client begins by encoding its measurements as an
-input for the PPM scheme and splitting it into input shares. (Note that the
-structure of each input share depends on the PPM scheme in use, its parameters,
-and the role of aggregator, i.e., whether the aggregator is a leader or helper.)
-To encrypt an input share, the client first generates an HPKE
+To generate the report, the client begins by running the input-distribution
```

Comment:
This kind of comes out of nowhere. I think you need a section describing the functions in the VDAF in overview.